src/App.jsx

```diff
@@ -141,9 +184,9 @@ const App = () => {
     alert(`Time set for this task:${taskTime}`);
```

Comment:
Be sparing in the use of alerts. They look like a handy notification system but alerts block the main thread. It's not going to cause you any problems here so there's no need to make any changes.

The usual practice is to set up a notification system that uses HTML elements. I like https://www.npmjs.com/package/react-toastify for this sort of thing.